sdk/storage/examples/table_00.rs

```diff
@@ -0,0 +1,111 @@
+use azure_core::prelude::*;
+use azure_storage::table::prelude::*;
+use azure_storage::{
+    clients::{AsStorageClient, StorageAccountClient},
+    table::clients::AsTableClient,
+};
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use std::sync::Arc;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct MyEntity {
+    #[serde(rename = "PartitionKey")]
+    pub city: String,
+    pub name: String,
+    #[serde(rename = "RowKey")]
+    pub surname: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
+    // First we retrieve the account name and master key from environment variables.
+    let account =
+        std::env::var("STORAGE_ACCOUNT").expect("Set env variable STORAGE_ACCOUNT first!");
+    let master_key =
+        std::env::var("STORAGE_MASTER_KEY").expect("Set env variable STORAGE_MASTER_KEY first!");
+
+    let table_name = std::env::args()
+        .nth(1)
+        .expect("please specify the table name as first command line parameter");
+
+    let http_client: Arc<Box<dyn HttpClient>> = Arc::new(Box::new(reqwest::Client::new()));
+
+    let storage_account_client =
+        StorageAccountClient::new_access_key(http_client.clone(), &account, &master_key);
+
+    let table_service = storage_account_client
+        .as_storage_client()
+        .as_table_service_client()?;
+
+    let table = table_service.as_table_client(table_name);
+    let response = table.create().execute().await?;
+    println!("response = {:?}\n", response);
+
+    let mut entity = MyEntity {
+        city: "Milan".to_owned(),
+        name: "Francesco".to_owned(),
+        surname: "Cogno".to_owned(),
+    };
+
+    let partition_key_client = table.as_partition_key_client(&entity.city);
+
+    let mut transaction = Transaction::default();
+
+    transaction.add(table.insert().to_transaction_operation(&entity)?);
+
+    entity.surname = "Doe".to_owned();
+    transaction.add(table.insert().to_transaction_operation(&entity)?);
+
+    entity.surname = "Karl".to_owned();
+    transaction.add(table.insert().to_transaction_operation(&entity)?);
+
+    entity.surname = "Potter".to_owned();
+    let entity_client = partition_key_client.as_entity_client(&entity.surname)?;
+    transaction.add(
+        entity_client
+            .insert_or_replace()
+            .to_transaction_operation(&entity)?,
+    );
+
+    let response = partition_key_client
+        .submit_transaction()
+        .execute(&transaction)
+        .await?;
+    println!("response = {:?}\n", response);
+
+    //let response = table.insert().return_entity(false).execute(&entity).await?;
```

Comment:
I'd prefer to remove this commented out code. If there's an issue we should make a GitHub issue for it and track it there. Comments like this are confusing.

Reply:
Yes it was valid code I commented out while I was working on the transaction code. And of course I forgot to uncomment it out 🤦‍♂️ . Sorry about that!